lib/ansible/modules/user.py

```diff
@@ -1169,6 +1184,8 @@ def ssh_key_gen(self):
             else:
                 return (None, 'Key already exists, use "force: yes" to overwrite', '')
         cmd = [self.module.get_bin_path('ssh-keygen', True)]
+        cmd.append('-a')
```

Comment:
This appears to be implemented for Alpine Linux only, this should be moved to the base class.

Reply:
I missed this earlier, this module uses several classes to create a 'per system' object to handle the actions, you only updated the class for Alpine (which is derived from busybox, which is derived from user).

We need to ensure that all variants get the update.

Reply:
I've added an integration test called "test_ssh_key_rounds.yml". It creates an SSH key with 500 rounds. I'm not exactly sure but for me it looks like this test runs on all containers not only on Alpine.